build/package/Dockerfile.start

```diff
@@ -17,6 +18,14 @@ RUN mkdir -p $TEKTONCD_PATH && \
 
 WORKDIR $TEKTONCD_PATH/pipeline
 
+RUN cd /tmp \
+    && mkdir -p /tmp/git-lfs \
+    && curl -LO https://github.com/git-lfs/git-lfs/releases/download/v${GIT_LFS_VERSION}/git-lfs-linux-amd64-v${GIT_LFS_VERSION}.tar.gz \
```

Comment:
Caution, we should cater for arm as well to path the way for usage on M1 ;)

Reply:
I thought about that, I will catch up with @henrjk